Update week-02-training-plan.md (#15)
open-life-science/open-seeds-self-organised: week-02/week-02-training-plan.md

> # Week 02

Reviewer: ```suggestion
(week-02-training-plan)=
# Week-02 Training Plan
```

**Theme: Tooling for Project Design**

Download all presentations and assignments for week 02: [https://doi.org/10.5281/zenodo.14268572](https://doi.org/10.5281/zenodo.14268572) 

> *When reusing, please cite this resource as: OLS Community (2024). Open Seeds OLS-9 Week-2 Content \- Tooling for Design (open leadership, open canvas, road mapping). Zenodo. [https://doi.org/10.5281/zenodo.14268572](https://doi.org/10.5281/zenodo.14268572)* 

Reviewer: ```suggestion
*When reusing, please cite this resource as: OLS Community (2024). Open Seeds OLS-9 Week-2 Content - Tooling for Design (open leadership, open canvas, road mapping). Zenodo. [https://doi.org/10.5281/zenodo.14268572](https://doi.org/10.5281/zenodo.14268572)* 

```

## Week 02 Objectives:

* Identify tools to help you design your project in a clear way for newcomers to quickly understand what the project is about, who can be involved, and where the work is headed  
* Fill the open canvas for your project to think through an open project strategy to develop an MVP  
* Reflect on the roadmap for your project with milestones \- start drafting them with descriptions and tasks  
* After this Week, develop documentation and resources for your project and share them with others

   
## Before you get started:

* Complete the self-assessment exercise and draft a vision statement for your project (assignments from the previous week)

## What to expect in week 02?

* Identify ways for your team members and new colleagues to quickly understand what the project is about, who can be involved, and where the work is headed  
* Learn about “open canvas” and learn how to use it for your project in thinking through an open project strategy to develop an MVP  
* Reflect on the roadmap for your project with milestones \- start drafting them with descriptions and tasks

> ## Learning plan for this week 02:The learning plans for [self learning Individual](https://github.com/open-life-science/open-seeds-self-organised/blob/main/week-02/week-02-self-learning.md), for [group/cohort learning](https://github.com/open-life-science/open-seeds-self-organised/blob/main/week-02/week-02-group-learning.md)

Reviewer: ```suggestion
## Learning plan for this week 02:

* For self-paced learning, please create a copy of {ref}`Week 02 Self-Paced Template<week-02-self-learning>`
* For self-paced learning, please create a copy of {ref}`Week 02 Group-Based Template<week-02-group-learning.md>`

```

Reviewer: Since we are using Jupyter Book, we will have to use {ref} for cross referencing within the book. This is why we are creating labels on the top of each file.

Reviewer: Please can you follow this format for the next weeks too?

## Slides to link in Week 02 Notes:

> * Open Science for Open Leadership: [https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20open%20leadership.pdf?download=1](https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20open%20leadership.pdf?download=1)   
> * Introducing Tooling for Project Design: [https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20Tooling%20for%20Project%20Design.pptx?download=1](https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20Tooling%20for%20Project%20Design.pptx?download=1)   
> * Introducing Open Canvas: [https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20Open%20Canvas%20for%20Project%20Strategy.pptx?download=1](https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20Open%20Canvas%20for%20Project%20Strategy.pptx?download=1)   
> * Introducing Project road mapping: [https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20Roadmapping.pptx?download=1](https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20Roadmapping.pptx?download=1) 

Reviewer: ```suggestion
* Open Science for Open Leadership: [Slide Deck on Zenodo](https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20open%20leadership.pdf?download=1)   
* Introducing Tooling for Project Design: [Slide Deck on Zenodo](https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20Tooling%20for%20Project%20Design.pptx?download=1)   
* Introducing Open Canvas: [Slide Deck on Zenodo](https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20Open%20Canvas%20for%20Project%20Strategy.pptx?download=1)   
* Introducing Project road mapping: [Slide Deck on Zenodo](https://zenodo.org/records/14268572/files/%5BOLS-9%5D%20Week%2002%20-%20Roadmapping.pptx?download=1) 

```

## Assignments after you have completed the learning plan for week 02:

> Create these resources for your project:
> 
> 1. Start working on your Open Canvas: [Template, instruction and example](https://zenodo.org/records/14268572/files/%5BMAKE%20A%20COPY%5D%20%5BOLS-9%5D%20Week%2002%20-%20Open%20Canvas%20-%20Template.pptx?download=1%20) (make a personal copy)  
> 2. Start your Roadmap and comment on your issue with your draft Roadmap: [Template Link](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/) (make a personal copy)  
> 3. Complete this compare and contrast assignment about current and desired community interactions and value exchanges: [Template Link](https://zenodo.org/records/14268572/files/%5BMAKE%20A%20COPY%5D%20Week%2002%20-%20Compare%26Contrast.docx?download=1%20) (make a personal copy)

Reviewer: ```suggestion
Please follow the resource provided in the {ref}`Week 02 Assignment Page<week-02-assignments>`.

```
